dcicutils/ff_utils.py

```diff
@@ -485,14 +485,15 @@ def get_item_facets(item_type, key=None, ff_env=None):
     return facets
 
 
-def get_item_facet_values(item_type, key=None, ff_env=None):
+def get_search_facet_values(search_query, key=None, ff_env=None):
```

Comment:
Should the `search_query` not maybe default more easily?

Untested:
```suggestion
DEFAULT_SEARCH_QUERY_PATTERN = "search/?type=%s"


def get_search_facet_values(search_query=None, item=None, key=None, ff_env=None):
    if (not search_query and not item) or (search_query and item):
        raise ValueError("You must supply exactly one of 'search_query' or 'item')
    elif not search_query:
        search_query = DEFAULT_SEARCH_QUERY_PATTERN % item
```